scrapy_poet/injection.py

```diff
@@ -61,6 +66,17 @@ def load_providers(self, default_providers: Optional[Mapping] = None):
         self.is_class_provided_by_any_provider = \
             is_class_provided_by_any_provider_fn(self.providers)
 
+    def init_cache(self):
+        self.cache = None
+        cache_filename = self.spider.settings.get('SCRAPY_POET_CACHE')
+        if cache_filename and isinstance(cache_filename, bool):
```

Comment:
If the value is `True` (a boolean), then the cache location is automatic: it will be under `.scrapy` folder. If the value is a folder (that is, of type `str`) then the cache will be saved in the corresponding folder. If false or empty, then the cache will be disabled.